src/libs/antares/study/scenario-builder/rules.cpp

```diff
@@ -156,25 +156,27 @@ void Rules::readThermalCluster(const AreaName::Vector& splitKey, String value, b
         {
             string clusterId = (area->id).to<string>() + "." + clustername.to<string>();
             disabledClustersOnRuleActive[clusterId].push_back(year);
+            return false;
```

Comment:
I don't think we should return in this case. This error is not fatal, it only gives a warning message.

Reply:
In other words, this line makes a non-fatal error become fatal. This is probably not what we want.

Reply:
Fortunately, this error is not fatal, as the caller of **readLine** function (see **Sets::internalLoadFromINIFile(...)**) does not use its returned status.